supabase/functions/og-images/handler.tsx

```diff
@@ -30,7 +30,7 @@ export async function handler(req: Request) {
 
   switch (site) {
     case 'docs':
-      return new ImageResponse(( <Docs title={title} description={description} type={type} icon={icon} /> ),
+      return new ImageResponse(( <Docs title={title} description={(description !=='undefined')? description:''} type={type} icon={icon} /> ),
```

Comment:
```suggestion
      return new ImageResponse(( <Docs title={title} description={(description || ''} type={type} icon={icon} /> ),
```

Reply:
Hi, issue persists with the suggested change.

<img width="1512" alt="Screenshot 2023-05-10 at 7 52 13 AM" src="https://github.com/supabase/supabase/assets/73219529/aa5bcc37-8c52-4e1c-a6c9-1a6494d20650">

Reply:
Ah, never mind, I assumed description to be undefined and not the string 'undefined'

Reply:
Curious if you know where the string `'undefined'` is coming from? Is there a place up the chain we should be addressing the fix?

Reply:
Hi, It's coming in directly as a description query parameter. I'm not exactly sure why the description 'undefined' needs to be handled specifically. Perhaps @Isaiah-Hamilton can provide the context.

Reply:
Got it - must be on the Next.js SSG/SSR side. We can merge this as is and look into that at a later time.